packages/experimental/Icon/src/Icon.tsx

```diff
@@ -0,0 +1,109 @@
+import * as React from 'react';
+import {IconProps, SvgIconProps, FontIconProps} from './Icon.types';
+import * as ReactNative from 'react-native';
+import { Text } from '@fluentui-react-native/text';
+import {SvgUri} from 'react-native-svg'
+import * as assetRegistry from 'react-native/Libraries/Image/AssetRegistry';
+import { stagedComponent, useTheme, mergeProps, getMemoCache } from '@fluentui-react-native/framework';
+
+const rasterImageStyleCache = getMemoCache<ReactNative.ImageStyle>();
+
+function renderRasterImage(iconProps: IconProps) {
+  const { width, height } = iconProps;
+  const style = rasterImageStyleCache({
+    width: width,
+    height:height
+  }, [width, height])[0];
+
+  return (
```

Comment:
You probably want to forward the style from icon props here, which means you'll have to merge it with your style. I believe the merge-props package has an exported mergeStyles (which is what mergeProps uses internally), so:

```ts
const style = mergeStyles(props.style, rasterImageStyleCache({ width, height }, [width, height])[0]);
```